### DISCUSSION TITLE
Reconsider cache download buffer size

### DISCUSSION THREAD

**Discussion by stucka** [biglocalnews/clean-scraper#80] [Ideas]
cache.download now grabs 8k at a time; 1024*1024, a megabyte, may be more performant.

**Comment by newsroomdev** [comment]
Not a bad idea. San Diego is running `cache.download` in roughly 5mins. We can revisit later once we're given the all-clear that we can run additional scrapers in production. I am grabbing a megabyte at a time in `clean-prefect/flows/assets.py`'s streaming download, so parity would be nice if we need to debug
